src/sentry/constants.py

```diff
@@ -37,7 +37,7 @@ def get_all_languages():
 MODULE_ROOT = os.path.dirname(__import__('sentry').__file__)
 DATA_ROOT = os.path.join(MODULE_ROOT, 'data')
 
-VERSION_LENGTH = 250
+VERSION_LENGTH = 200
```

Comment:
Is this constant used anywhere else where a change might cause problems?

Reply:
This will definitely cause a problem now.

Reply:
@mattrobenolt and @alex-hofsteede I should have taken care of that now. I separate out the version_length that the rest of the application will use with the Release's model's column size. Let me know what you think!